aodncore/pipeline/steps/harvest.py

```diff
@@ -76,7 +76,7 @@ def run(self, pipeline_files):
                 if not deletion_map:
                     self._logger.info('No files to delete, proceeding to add matching files')
                 else:
-                    self.run_deletions(deletion_map, self.tmp_base_dir)
+                    self.run_deletions(deletion_map, self.tmp_base_dir, 'pending_store_deletion')
```

Comment:
Get rid of last attribute since we don't need it anymore once the undo method is totally separate